docs/recipes/sourcemaps-with-sass-and-autoprefixer.md

```diff
@@ -0,0 +1,37 @@
+# Sourcemaps with Sass (gulp-sass) and Autoprefixer
+
+Sass and Autoprefixer play well together, but it can be hard to get nice sourcemaps when mixing these. 
+This solution seems to work for many people.
+
+
+```js
+// npm install --save-dev gulp gulp-autoprefixer gulp-sass gulp-sourcemaps gulp-filter gulp-plumber
+
+var gulp = require('gulp');
+var autoprefixer = require('gulp-autoprefixer');
+var sass = require('gulp-sass');
+var sourcemaps = require('gulp-sourcemaps');
+var gulpFilter = require('gulp-filter');
```

Comment:
filter is a much more robust API and allows you to use many steps between a single filter/restore.  when you are using a single step between, it is cleaner and preferred to use gulp-if.